app/models/oral_history_content/interviewee_honor.rb

```diff
@@ -0,0 +1,14 @@
+require 'attr_json'
+class OralHistoryContent
+  class IntervieweeHonor
+    include AttrJson::Model
+    validates_presence_of :date
+    validates_presence_of :honor
+
+    validates_format_of :date, with: /\A\d{4}(-\d{2}(-\d{2})?)?\z/,
+      message: "must be of format YYYY[-MM-DD]"
```

Comment:
We're re-using this date format validation in a bunch of places, including originally in Work::DateOfWork. Let's find some way to DRY it, the regexp and validation failure message?

Reply:
Created a special validator, but decided against using it for DateOfWork, at least for now.